renovate.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "prConcurrentLimit": 2,
```

Comment:
Each of these individual config options is detailed here: https://renovatebot.com/docs/configuration-options/